docs/recipes/git-show.rst

```diff
@@ -40,6 +40,22 @@ Show all files in commit
     >>> for e in commit.tree:
     >>>     print(e.name)
 
+======================================================================
+Produce something like a `git show` message
+======================================================================
+
+    >>> from __future__ import unicode_literals
+    >>> from datetime import datetime
+    >>> import pytz
+    >>> tzinfo  = pytz.timezone('Europe/Berlin')
+    >>> dt      = datetime.fromtimestamp(float(commit.commit_time), tzinfo)
```

Comment:
git-show/git-log use the author time rather than commit time. In order to show what it would show, we'd want to use `commit.author.time` here.